### PR TITLE
Add RELEASING.md

### PR DESCRIPTION
**Pull request by pschorf**
## Changes proposed in this PR
- Add a RELEASING.md doc for cook executor

## Why are we making these changes?
Describe the process for releasing a new executor version to PyPI

### CODE REVIEW

File: executor/RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+------------
+Since PyPI does not allow modifying releases, it can be useful to test a release using their test instance.
+```bash
+rm -rf dist/*
+python3 setup.py sdist bdist_wheel
+python3 -m twine upload --repository-url https://test.pypi.org/legacy/ dist/*
+```
+Then, in a separate virtualenv for testing:
+```bash
+pip install  --index-url https://test.pypi.org/simple/ --no-deps cook-executor==$VERSION
```
Comment (shamsimam) [inline executor/RELEASING.md]: `pip3`?

File: executor/RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+pip install pymesos==0.3.9 # install any other required dependencies from the main pypi repo
+cook-executor
+```
+If there is an issue with the release, you can just release another version. They are GCed periodically from the test instance.
```
Comment (shamsimam) [inline executor/RELEASING.md]:
```suggestion
If there is an issue with the release, you can just release another version. They are GC-ed periodically from the test instance.
```

File: executor/RELEASING.md (hunk 1 above)
Comment (pschorf) [reply executor/RELEASING.md]: Done

File: executor/RELEASING.md (hunk 2 above)
Comment (pschorf) [reply executor/RELEASING.md]: Done

**Pull request merged**